commandapi-nms/commandapi-1.19-common/src/main/java/dev/jorel/commandapi/nms/NMS_1_19_Common.java

```diff
@@ -577,6 +536,111 @@ public final ParticleData<?> getParticle(CommandContext<CommandSourceStack> cmdC
 		return new ParticleData<Void>(particle, null);
 	}
 
+	/**
+	 * {@link #getParticleDataAsDustColorTransitionOption(Particle, DustColorTransitionOptions) Parent Method}
+	 * @param particle
+	 * @param options
+	 * @return
+	 */
+	private ParticleData<DustTransition> getParticleDataAsDustColorTransitionOption(Particle particle, DustColorTransitionOptions options) {
+		final Color color = Color.fromRGB((int) (options.getColor().x() * 255.0F),
+			(int) (options.getColor().y() * 255.0F), (int) (options.getColor().z() * 255.0F));
+		final Color toColor = Color.fromRGB((int) (options.getToColor().x() * 255.0F),
+			(int) (options.getToColor().y() * 255.0F), (int) (options.getToColor().z() * 255.0F));
+		return new ParticleData<>(particle, new DustTransition(color, toColor, options.getScale()));
+	}
+
+	/**
+	 * {@link #getParticle(CommandContext, String)}  Parent Method}
+	 * @param particle
+	 * @param options
+	 * @return
+	 */
+	private ParticleData<DustOptions> getParticleDataAsDustParticleOptions(Particle particle, DustParticleOptions options) {
```

Comment:
This method is really small, did it really need its own method for these two statements?